packages/manager/src/features/Linodes/LinodesDetail/LinodeConfigs/LinodeConfigDialog.styles.ts

```diff
@@ -0,0 +1,46 @@
+import { styled } from '@mui/material/styles';
+
+import { Divider } from 'src/components/Divider';
+import { FormControl } from 'src/components/FormControl';
+import { FormControlLabel } from 'src/components/FormControlLabel';
+import { FormGroup } from 'src/components/FormGroup';
+import { RadioGroup } from 'src/components/RadioGroup';
+
+const formGroupStyling = () => ({
+  '&.MuiFormGroup-root[role="radiogroup"]': {
+    marginBottom: 0,
+  },
+  alignItems: 'flex-start',
+});
```

Comment:
Does the need to be a function? 

```suggestion
const formGroupStyling = {
  '&.MuiFormGroup-root[role="radiogroup"]': {
    marginBottom: 0,
  },
  alignItems: 'flex-start',
};
```